guake/paths.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8; -*-
```

Comment:
normally you do not commit this file is automatically generated by the makefile

Reply:
Of course, you are right!
I rebased and removed the file.